Traceback (most recent call last):
  File "D:\programowanie\python\projekt zespołowy\struktura\architectures\ml\RandomForest\train.py", line 27, in train
    model.fit(X_train, y_train)
  File "D:\programowanie\python\projekt zespołowy\struktura\.venv\Lib\site-packages\sklearn\base.py", line 1467, in wrapper
    estimator._validate_params()
  File "D:\programowanie\python\projekt zespołowy\struktura\.venv\Lib\site-packages\sklearn\base.py", line 666, in _validate_params
    validate_parameter_constraints(
  File "D:\programowanie\python\projekt zespołowy\struktura\.venv\Lib\site-packages\sklearn\utils\_param_validation.py", line 95, in validate_parameter_constraints
    raise InvalidParameterError(
sklearn.utils._param_validation.InvalidParameterError: The 'max_features' parameter of RandomForestClassifier must be an int in the range [1, inf), a float in the range (0.0, 1.0], a str among {'log2', 'sqrt'} or None. Got 'auto' instead.